paddle/fluid/distributed/fleet_executor/CMakeLists.txt

```diff
@@ -5,7 +5,7 @@ endif()
 proto_library(interceptor_message_proto SRCS interceptor_message.proto)
 
 if(WITH_DISTRIBUTE AND NOT (WITH_ASCEND OR WITH_ASCEND_CL))
-  set(BRPC_DEPS brpc ssl crypto)
+  set(BRPC_DEPS brpc ssl crypto protobuf gflags glog zlib leveldb snappy gflags glog)
```

Comment:
Should these new dependencies be also added to the `BRPC_DEPS` below?

Reply:
yes，or make test failed